test/src/subnet/main.cpp

```diff
@@ -23,107 +23,143 @@
-using local_time=cadmium::logger::logger<cadmium::logger::logger_local_time, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using log_all=cadmium::logger::multilogger<info, debug, state, log_messages, routing, global_time, local_time>;
-
-using logger_top=cadmium::logger::multilogger<log_messages, global_time>;
-
-
-/*******************************************/
-
-
-
-/********************************************/
-/****** APPLICATION GENERATOR *******************/
-/********************************************/
-string input_data = "../test/data/subnet/subnet_input_test.txt";
-const char * i_input_data = input_data.c_str();
-
-std::shared_ptr<cadmium::dynamic::modeling::model> generator = cadmium::dynamic::translate::make_dynamic_atomic_model<ApplicationGen, TIME, const char* >("generator" , std::move(i_input_data));
-
-
-/********************************************/
-/****** SUBNET *******************/
-/********************************************/
-
-std::shared_ptr<cadmium::dynamic::modeling::model> subnet1 = cadmium::dynamic::translate::make_dynamic_atomic_model<Subnet, TIME>("subnet1");
+    using
+        info = cadmium::logger::logger<cadmium::logger::logger_info,
+            cadmium::dynamic::logger::formatter<TIME>,
+                oss_sink_provider>;
+    using
+        debug = cadmium::logger::logger<cadmium::logger::logger_debug,
+            cadmium::dynamic::logger::formatter<TIME>,
+                oss_sink_provider>;
+    using
+        state = cadmium::logger::logger<cadmium::logger::logger_state,
+            cadmium::dynamic::logger::formatter<TIME>,
+                oss_sink_provider>;
+    using
+        log_messages = cadmium::logger::logger<cadmium::logger::logger_messages,
+            cadmium::dynamic::logger::formatter<TIME>,
+                oss_sink_provider>;
+    using
+        routing = cadmium::logger::logger<cadmium::logger::logger_message_routing,
+            cadmium::dynamic::logger::formatter<TIME>,
+                oss_sink_provider>;
+    using
+        global_time = cadmium::logger::logger<cadmium::logger::logger_global_time,
+            cadmium::dynamic::logger::formatter<TIME>,
+                oss_sink_provider>;
+    using
+        local_time = cadmium::logger::logger<cadmium::logger::logger_local_time,
+            cadmium::dynamic::logger::formatter<TIME>,
+                oss_sink_provider>;
+    using
+        log_all = cadmium::logger::multilogger<info,
+            debug, state, log_messages, routing, global_time, local_time>;
+
+    using logger_top = cadmium::logger::multilogger<log_messages, global_time>;
+
+
+    /*******************************************/
+
+
+
+    /********************************************/
+    /****** APPLICATION GENERATOR *******************/
+    /********************************************/
+    string input_data =
+        "../test/data/subnet/subnet_input_test.txt";
+    const char * i_input_data = input_data.c_str();
```

Comment:
It would be const char* 
i.e.., there will be no space between char and *

Reply:
solved